### PR TITLE
Add variadic to param tag

### PR DESCRIPTION
**Pull request by jaapio**
PHP allows variadic parameters, and thus this shall be reflected in the tags.

### CODE REVIEW

File: proposed/phpdoc-tags.md [hunk 1]
```diff
@@ -368,7 +368,7 @@ Mark a function that has been generated using a PHP script.
 /**
  * @generated function generated using bin/script.php, please DO NOT EDIT!
  * 
- * @params array $params parameters for the index API endpoint
+ * @param array $params parameters for the index API endpoint
```
Comment (shadowhand) [inline proposed/phpdoc-tags.md:371]: Good catch. This is a typo and should be fixed separately.

File: proposed/phpdoc-tags.md [hunk 2]
```diff
@@ -604,6 +604,11 @@ single parameter of a function or method. When provided it MUST contain a
 @param tags are omitted due to all useful info already being visible in the
 code signature itself. The description is OPTIONAL yet RECOMMENDED.
 
+The name of the parameter MUST be prefixed with a dollar sign (`$`) to indicate
+the start of a variable name. OPTIONAL the variadic operator (`...`) MAY be used
```
Comment (samdark) [inline proposed/phpdoc-tags.md:608]:
```suggestion
the start of a variable name. OPTIONAL variadic operator (`...`) MAY be used
```

**Review by ashnazg — APPROVED** [review]

File: proposed/phpdoc-tags.md (hunk 1 above)
Comment (zhuo-xun-yi) [reply proposed/phpdoc-tags.md:371]: > 接得好。這是一個拼字錯誤，應單獨修復。

**Review by zhuo-xun-yi — COMMENTED** [review]
#1300

**Comment by zhuo-xun-yi** [comment]
> [第1300章](https://github.com/php-fig/fig-standards/pull/1300)

**Pull request merged**